fe/src/main/java/org/apache/doris/optimizer/OptGroup.java

```diff
@@ -42,4 +51,32 @@ public void addMExpr(MultiExpression mExpr) {
     public String debugString() {
         return "";
     }
+
+    public boolean isImplemented() {
```

Comment:
setter and getter should be put one line, this can make more code shown in one screen